### ISSUE TITLE
How to close contex menu when scrolling

### ISSUE THREAD

**Issue by NemanjaS2P** [fkhadra/react-contexify#161]
if I click on a page and the contex menu opens i want it to close when i scroll the page. Is there a property to set where i hide the menu on scroll?
Thank you

**Comment by fkhadra** [comment]
Hey @NemanjaS2P, this is already the default behavior. 

https://user-images.githubusercontent.com/5574267/102987372-7c2d3f00-4512-11eb-8189-a19f7db10b9a.mov

If it's not your case, could you share the code in a code sanbox ? Thanks

**Comment by codingbycat** [comment]
> Hey @NemanjaS2P, this is already the default behavior.
> 
>  Screen.Recording.2020-12-23.at.11.31.02.AM.mov 
> If it's not your case, could you share the code in a code sanbox ? Thanks

Open a custom scroll area to close the context-menu？
It can't seem to work well in custom scroll area.
See this code sanbox example: https://codesandbox.io/s/nifty-fermat-ljldf?file=/src/App.tsx:348-359

**Comment by aditya-padhi-kbl** [comment]
> > Hey @NemanjaS2P, this is already the default behavior.
> > Screen.Recording.2020-12-23.at.11.31.02.AM.mov
> > If it's not your case, could you share the code in a code sanbox ? Thanks
> 
> Open a custom scroll area to close the context-menu？
> It can't seem to work well in custom scroll area.
> See this code sanbox example: https://codesandbox.io/s/nifty-fermat-ljldf?file=/src/App.tsx:348-359

Yes I am able to reproduce this as well.  We can make it sticky with scroll or it should have an option to automatically hide it.

**Comment by ShahriarKh** [comment]
I have the same problem. The property that causes this problem is:
```css
height:calc(100vh - 52px);
```
and this is the full class applied on my page:
```css
{
  position: relative;
  overflow-y: scroll;
  overflow-x: hidden;
  height: calc(100vh - 52px);
  transition: all 0.5s ease-in-out;
}
```

**Comment by fkhadra** [comment]
As a workaround you could close the menu programmatically, for example, you can listen for the scroll on the target and then close the menu

```ts
target.addEventListener("scroll", () => contextMenu.hideAll())

```
